server/service/mdm.go

```diff
@@ -1437,7 +1437,7 @@ type batchSetMDMProfilesRequest struct {
 	TeamID        *uint                        `json:"-" query:"team_id,optional"`
 	TeamName      *string                      `json:"-" query:"team_name,optional"`
 	DryRun        bool                         `json:"-" query:"dry_run,optional"`        // if true, apply validation but do not save changes
-	AssumeEnabled bool                         `json:"-" query:"assume_enabled,optional"` // if true, assume MDM is enabled
+	AssumeEnabled *bool                        `json:"-" query:"assume_enabled,optional"` // if true, assume MDM is enabled
```

Comment:
Updated the setting so now we can assume disabled if `AssumeEnabled != nil`

Reply:
The default value is `appCfg.MDM.WindowsEnabledAndConfigured`

With this change, if the global config disables Windows MDM, and a team tries to upload a profile, we will fail during dry run.

Reply:
if the global config disables Windows MDM would the config be `false` or `nil`?  Maybe I'm confused what `AssumeEnabled == nil` means.